lib/containers/pr-changed-files-container.js

```diff
@@ -59,7 +59,16 @@ export default class PullRequestChangedFilesContainer extends React.Component {
   }
 
   buildPatch(rawDiff) {
-    const diffs = parseDiff(rawDiff);
+    const diffs = parseDiff(rawDiff).map(diff => {
```

Comment:
do we also need to worry about where we're using `parseDiff` in `git-shell-out-strategy`?  If so, we should consider extracting this functionality to some kind of helper.

I seem to remember some chatter that git gives us different output than the REST api does, so this might not actually be a problem.

Reply:
yes

Reply:
>I seem to remember some chatter that git gives us different output than the REST api does,

Indeed! It is a git diff default behaviour to use `a/` to prefix the source file and `b/` to prefix the destination file. In `git-shell-out-strategy`, we work around this by using `--no-prefix` flag to get diffs, but for changed files tab, we don't have that option since we're getting the diff from the REST API.